resources/views/front-page.blade.php

```diff
@@ -2,13 +2,13 @@
 
 @section('content')
 	<span itemscope itemtype="http://schema.org/ItemPage">
-		<div class="row mt-sm-4">
+        <div class="row mt-sm-4">
```

Comment:
somewhere in `front-page.blade.php` add `@include('partials.front-page-hero')` (likely not in a col-md-4, for instance)